example/Benchmark_spiner.cpp

```diff
@@ -0,0 +1,380 @@
+/*
```

Comment:
See my comment above about copyright. Also, I think these should be renamed to `benchmark_spiner_kokkos` and `benchmark_spiner_cpu` or something like that.